includes/classes/wp-maintenance-mode-admin.php

```diff
@@ -647,7 +647,7 @@ public function insert_template() {
 				$post_arr['ID'] = $this->plugin_settings['design']['page_id'];
 				$page_id        = wp_update_post( $post_arr );
 			} else {
-				$post_arr['post_title'] = 'Maintenance Page';
+				$post_arr['post_title'] = __( 'New Page', 'wp-maintenance-mode' );
```

Comment:
we can still use the `Maintenance Page` as default, translated. I think is more straightforward than New Page, even if the user selected a different category.